src/components/Header/Header.css

```diff
@@ -5,20 +5,24 @@
     color: white;
     text-shadow: -2px 0 black, 0 2px black,2px 0 black, 0 -2px black;
     font-size: 40px;
+    margin-left: 10px;
     position: relative;
+    top: 15px;
 }
 
 .logo {
     float: center;
     width: 20%;
+    height: 40px;
+    padding-top: 10px;
   }
 
   .header {
     position: fixed;
     top: 0;
     left: 0;
     width: 100%;
-    z-index: 999;
```

Comment:
should be able to merge this to main if you can add the z-index back! without it, i think the buttons float over the header when users scroll